src/components/Redemptions/RedemptionsPage.tsx

```diff
@@ -40,7 +40,43 @@ const mapDispatchToProps = {
   showNotification,
 };
 
-type IProps = IStateProps & IDispatchProps & ISubscriptionProps<[IDAOState[], any[]]>
+type IProps = IStateProps & IDispatchProps & ISubscriptionProps<any[]>
+
+// TODO: this should really be in the client library somehow
+const createDaoState = (data: any): IDAOState => {
```

Comment:
why? its just from an object with the dao data from the query, its not like you always name every constructor createFromQuery or createFromData. i could call it constructDaoState? but you are just passing in the data that is needed to construct an IDaoState

Reply:
I interpret this as a function that converts the input object to an `IDaoState`.  The proposed name better documents what it is doing.

Reply:
but what is the proposed name? createDaoStateFrom[what?] what is what? its just data from a query

Reply:
"[what]" is the name of whatever you're creating the DaoState from.  If you create an interface to describe the data then it could be the name of the interface.